README.md

```diff
@@ -1,19 +1,31 @@
-# Stereum 2.0
+![GitHub - Stereum 2 0 - ReadMe](https://user-images.githubusercontent.com/82385103/175423093-03d28d7b-3c2e-44b7-8be1-7f2a5946fcc6.png)
+
+---
 
 [![Support Server](https://img.shields.io/badge/Discord-Stereum-blue?style=flat-square)](https://discord.gg/8Znj8K6GjN) [![Support Email](https://img.shields.io/badge/Email-Stereum-brightgreen?style=flat-square)](mailto:stereum@stereum.net)
 
-This repository contains the code and scripts for ~~[Stereum 2.0](https://stereum.net/ethereum-node/)~~ (tba), an [Ethereum](https://ethereum.org/) node setup and management software with focus on self sovereignity, privacy and flexibility.
+---
+This repository contains the code and scripts for Stereum 2.0 (tba)
+
+
+🍄 Stereum is a tool to manage the process of setting up & maintaining an Ethereum node for you with a heavy focus on self sovereignty, privacy and flexibility. Stereum is actively developed by the RockLogic GmbH, who first started to develop on its predecessor in 2019.
```

Comment:
`Stereum manages the process of setting up....` sounds better to me, also `...with focus on self...`

`Stereum is actively developed by RockLogic GmbH` and link `RockLogic GmbH` to rocklogic.at? As far as I know `who` only references individuals, but I'm no native speaker 🤔

Reply:
Who is based on if it is a person or a thing & can be used for groups comprised of people: 
e.g. "The Romans, who are responsible for the fall of Carthage"

Though I agree reading it again. Sounds kind off sloppy.